src/main/java/com/google/devtools/build/lib/bazel/debug/workspace_log.proto

```diff
@@ -77,11 +77,17 @@ message FileEvent {
   // Path to the created file
   string path = 1;
   // Content of the created file
-  string content = 2;
+  bytes content = 4;
```

Comment:
Removing a field is a breaking change to the protocol.
Instead, let's have *two* fields for now: `string content` and `bytes bytes`, deprecating the first one. As part of an incompatible change to make legecy_utf8 default to false, we can set content to empty string and then remove that field.

Reply:
After further thought, I've reverted this message to only containing `string content`. The protobuf field has always been set from the `String content` directly, without encoding, so it doesn't make sense for it to change as part of this PR.

It might be reasonable to add a new field `bytes encoded_content` that differs depending on `legacy_utf8` mode, but I think that is best done separately.